aiounifi/interfaces/connectivity.py

```diff
@@ -156,6 +156,6 @@ async def _request(
 
 def _raise_on_error(data: "TypedApiResponse") -> None:
     """Check response for error message."""
-    if "meta" in data and data["meta"]["rc"] == "error":
+    if "meta" in data and data["meta"] is not None and data["meta"]["rc"] == "error":
         LOGGER.error(data)
         raise_error(data["meta"]["msg"])
```

Comment:
I was thinking about that method the other day, do you have any examples of errors from the v2 api? We should structure this in a scalable way.

Reply:
`{"errorCode":400,"message":"updateTrafficRule.arg2.matchingTarget: must not be null, updateTrafficRule.arg2.action: must not be null, updateTrafficRule.arg2.targetDevices: must not be empty"}`

This is the error I get for example when putting a wrong value to the endpoint for toggling a traffic rule. This is the 'raw' response, so that's everything we got to work with.

Reply:
After doing the other modifications, the `and data["meta"] is not None` part is no longer needed.
But we still need to find a way to handle errors of the v2 API.

Reply:
So suggestion is to keep the old logic as is and add a second if-statement that manages your data.